containers/npm_packages.go

```diff
@@ -14,10 +15,13 @@ func NPMPackages(d *dagger.Client, platform dagger.Platform, src *dagger.Directo
 
 	c = WithYarnCache(c, opts)
 
+	ersion := strings.TrimPrefix(version, "v")
+
 	c = c.WithExec([]string{"mkdir", "npm-packages"}).
 		WithExec([]string{"yarn", "install", "--immutable"}).
 		WithExec([]string{"yarn", "run", "packages:build"}).
-		// TODO: We should probably start reusing the yarn pnp map if we can figure that out isntead of rerunning yarn install everywhere.
+		// TODO: We should probably start reusing the yarn pnp map if we can figure that out instead of rerunning yarn install everywhere.
+		WithExec([]string{"yarn", "run", "lerna", "version", ersion, "--exact", "--no-git-tag-version", "--no-push", "--force-publish", "-y"}).
```

Comment:
You have to do a yarn install to update the lockfile at this point I believe, because if you don't, then later steps that do `yarn install --immutable` will fail

Reply:
I don't think there are any later steps that do `yarn install --immutable`, should we run `yarn install --mode=update-lockfile` after updating the version?

Reply:
hmmmmm. Since it's containerized and not using a shared volume I actually don't know if doing that would update it for everything afterwards without exporting that file so maybe for now we should just leave it?

Reply:
if it becomes a problem later then i guess we can resolve it at that point